lib/puppet/ssl/host.rb

```diff
@@ -484,6 +480,20 @@ def load_certificate_bundle(bundle_string)
     end
   end
 
+  # Fetches the crl bundle from the CA server
+  # @raise [Puppet::Error] if response from the server is not a valid certificate
+  #                        bundle
+  # @return [[OpenSSL::X509::CRL]] the certs loaded from the response
+  def download_crl_bundle
+    http_client.ensure_verify_peer_connection_mode
```

Comment:
I think we need to set CRL checking to false for the cert store? I don't remember where we landed on that. There was also talk of explicitly configuring the verify callback to check or not check CRLs.